tests/excerptexport/management/test_purge_old_result_files.py

```diff
@@ -24,6 +24,21 @@ def output_file_with_file_too_old(db, output_file_with_file, mocker):
     return output_file_with_file
 
 
+@pytest.fixture
+def output_file_too_old(db, output_file, mocker):
```

Comment:
"too old" for what? Also, this fixture provides a file (model) object, not a boolean, so the name should _end_ in "file". Maybe `expired_file`?